src/test/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConnectorIntegrationTest.java

```diff
@@ -0,0 +1,116 @@
+package com.marklogic.kafka.connect.sink;
+
+import com.marklogic.client.io.SearchHandle;
+import com.marklogic.client.query.DeleteQueryDefinition;
+import com.marklogic.client.query.StructuredQueryBuilder;
+import com.marklogic.client.query.StructuredQueryDefinition;
+import com.marklogic.junit5.spring.SimpleTestConfig;
+import kafka.server.KafkaConfig$;
+import net.mguenther.kafka.junit.*;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import static net.mguenther.kafka.junit.EmbeddedConnectConfig.kafkaConnect;
+import static net.mguenther.kafka.junit.EmbeddedKafkaCluster.provisionWith;
+import static net.mguenther.kafka.junit.EmbeddedKafkaClusterConfig.newClusterConfig;
+import static net.mguenther.kafka.junit.ObserveKeyValues.on;
+
+public class MarkLogicSinkConnectorIntegrationTest extends AbstractIntegrationTest {
+
+    // Declared by AbstractSpringMarkLogicTest
+    @Autowired
+    private SimpleTestConfig testConfig;
+
+    private final String ML_COLLECTION = "kafka-data";
+    private final String TOPIC = "test-topic";
+    private final String KEY = String.format("key-%s", UUID.randomUUID());
+
+    private EmbeddedKafkaCluster kafka;
+
+    @BeforeEach
+    void setupKafka() {
+        provisionKafkaWithConnectAndMarkLogicConnector();
+        kafka.start();
```

Comment:
This results in a bunch of stacktraces about optional dependencies not existing. I think we can safely ignore those, and this does the trick in logback.xml:

```
<logger name="org.reflections" level="ERROR" additivity="false">
    <appender-ref ref="STDOUT"/>
  </logger>
```